.gitignore

```diff
@@ -165,3 +167,7 @@ $RECYCLE.BIN/
 
 # Mac desktop service store files
 .DS_Store
+
+#ignore js
+sample_angular2/src/wwwroot/app/*.js
```

Comment:
those are now stale, should be changed to Samples/resource-owner-password-credential/Angular2/src/wwwroot/app/*.js